core/common/src/main/java/alluxio/exception/runtime/AlluxioRuntimeException.java

```diff
@@ -97,7 +103,15 @@ public StatusException toGrpcStatusException() {
         trailers.put(ProtoUtils.keyForProto(Any.getDefaultInstance()), detail);
```

Comment:
Why are we having `trailers = new Metadata()` again? Won't this overwrite the retryable and error type info?